revscoring/dependencies/dependent.py

```diff
@@ -99,55 +99,60 @@ class DependentSet:
             A base name for the items in the set
     """
 
-    def __init__(self, name, _dependents=None, _dependent_sets=None):
-        self._dependents = _dependents or set()
-        self._dependent_sets = _dependent_sets or set()
-        self._name = name
+    def __init__(self, name, dependents=None, dependent_sets=None):
+        self.dependents = dependents or {}
+        self.dependent_sets = dependent_sets or {}
+        self.name = name
```

Comment:
This is good, but I believe you'll need to update the tests to use `name` instead of `_name` now.